digest/handler_operation.go

```diff
@@ -275,6 +276,18 @@ func (hd *Handlers) buildOperationHal(va OperationValue) (Hal, error) {
 	return hal, nil
 }
 
+func (hd *Handlers) buildNFTAgentHal(va collection.AgentBox, address base.Address, symbol string) (Hal, error) {
+	var hal Hal
+
+	h, err := hd.combineURL(HandlerPathAccountNFTAgent, "address", address.String(), "symbol", symbol)
+	if err != nil {
+		return nil, err
+	}
+	hal = NewBaseHal(va, NewHalLink(h, nil))
+
+	return hal, nil
+}
+
```

Comment:
@soonkuk 
I just wonder if there is any special reason that the declaration of `buildNFTAgentHal(...)` is in `handler_operation.go` instead of `handler_account.go`.

Reply:
@wyuinche 
Your are right.
It is better `buildNFTAgentHal` is in handler_account.go.
I fixed it.